lectures/eigen_II.md

```diff
@@ -485,7 +486,7 @@ The following is a fundamental result in functional analysis that generalizes
 
 Let $A$ be a square matrix and let $A^k$ be the $k$-th power of $A$.
 
-Let $r(A)$ be the dominant eigenvector or as it is commonly called the *spectral radius*, defined as $\max_i |\lambda_i|$, where
+Let $r(A)$ be the *spectral radius* of $A$, defined as $\max_i |\lambda_i|$, where
```

Comment:
Our style is to use **bold** for definitions: https://manual.quantecon.org/styleguide/writing.html#emphasis-and-definitions